redis_json/src/commands.rs

```diff
@@ -1740,22 +1696,11 @@ where
 pub fn json_clear<M: Manager>(manager: M, ctx: &Context, args: Vec<RedisString>) -> RedisResult {
     let mut args = args.into_iter().skip(1);
     let key = args.next_arg()?;
-    let paths = args.try_fold::<_, _, Result<Vec<Path>, RedisError>>(
```

Comment:
@iddm I believe this to be a bug. This `try_fold` implies that multiple paths may be passed to `JSON.CLEAR`, and that if, for *any* path arg, if `arg.try_as_str().is_err()` then the function shall return early. Even if there was a legal path that was passed. Whereas the [documentation](https://redis.io/docs/latest/commands/json.clear/) clearly states that, at most, one singular path may be passed.

If anything, I believe the args length should be checked, and if it is not correct, then return `Err(WrongArity)`.